src/api/.rubocop_todo.yml

```diff
@@ -815,6 +814,7 @@ RSpec/LetSetup:
     - 'spec/controllers/webui/request_controller_spec.rb'
     - 'spec/controllers/webui/search_controller_spec.rb'
     - 'spec/controllers/webui/staging/workflows_controller_spec.rb'
+    - 'spec/controllers/webui/users/notifications_controller_spec.rb'
```

Comment:
we should disable this cop. It's normal in our setup to have to create objects but don't call it directly in the spec.

Reply:
I agree. In this case, I don't think it worth it to try to find an alternative for all the tests affected. At least not in the near future.

Reply:
Done @vpereira